<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SUTVR</class>
 <widget class="QMainWindow" name="SUTVR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SUTVR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>Ожидание команды</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Задать</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Содержание неисправности</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="text">
        <string>Норма изделия 15Щ72/Неготовность изделия 15Щ72</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Неисправность вентиляционной системы 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Неисправность вентиляционной системы 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Температура вне диапазона допустимых значений</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Влажность вне диапазона допустимых значений</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Температура выше нормы</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="text">
        <string>Влажность выше нормы</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="firstPort">
      <property name="text">
       <string>Первый адрес</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QCheckBox" name="secondPort">
      <property name="text">
       <string>Второй адрес</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="console">
      <property name="text">
       <string>Вывести в консоль</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
